src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -1,28 +1,26 @@
-using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.IO;
-using System.Linq;
-using System.Security.Permissions;
-using System.Web.UI.WebControls;
-using System.Threading;
-using System.Windows;
-using System.Windows.Threading;
-using DesignScript.Builtin;
 using Dynamo.Configuration;
-using Dynamo.Core;
 using Dynamo.DocumentationBrowser.Properties;
 using Dynamo.Graph;
 using Dynamo.Graph.Annotations;
 using Dynamo.Graph.Workspaces;
 using Dynamo.Logging;
-using Dynamo.Selection;
 using Dynamo.Models;
-using DynamoProperties = Dynamo.Properties;
 using Dynamo.PackageManager;
+using Dynamo.Selection;
 using Dynamo.ViewModels;
 using Dynamo.Wpf.Extensions;
 using Dynamo.Wpf.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Security.Permissions;
+using System.Threading;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Threading;
+using DynamoProperties = Dynamo.Properties;
```

Comment:
Please follow https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#usings

Reply:
I did not know this setting, thanks.